### PR TITLE
Added some solutions

### PR DESCRIPTION
**Pull request by Edlingeer**

### CODE REVIEW

**Comment by DanielSank** [comment]
Thank you! I might not get around to this until the weekend, but in any case the contribution is very much appreciated.

**Comment by DanielSank** [comment]
I'm finally returning to this project. Yay. Should have this merged soon.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,8 @@
+# Compiled source #
 build
+main.aux
+main.log
+main.out
+main.pdf
+main.synctex.gz
+main.toc
```
Comment (DanielSank) [inline .gitignore]: FWIW I set my latex system to put built files in a `build/` directory (hence the top line of this file). No problem adding the other lines too, for folks who build in-place.

File: chapter2/Description_sequence_random_events.tex [hunk 2]
```diff
@@ -0,0 +1,24 @@
+\leveldown{Description of a sequence of random events - pg. 32}
+
+\leveldown{Problem}
+
+Calculate $Q_s$ for the given recipe of construction of a random set of dots
+\levelstay{Solution}
+We need to find a description of a sequence of random events at $\tau_1$, $\tau_2$, ... in terms of the probability distribution $Q_s$ using the probability $w$. The $w$ probability is the simplest description of correlated random events : the new arrival is dependent on the timing from the previous arrival and both arrivals are governed by the same probability function $w$. We may as well represent this situation by having $s$ replicas of the system and reporting the first random events in each of these systems.Such first event would be governed by a probability $w$. We can write $Q_s$ as a probability of events that are independent of one another in each of these systems.
```
Comment (DanielSank) [inline chapter2/Description_sequence_random_events.tex]: There are a few punctuation errors here, i.e. a space missing at the end of a sentence, etc.

File: chapter2/Mean_square_N_interval.tex [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+\end{split}
+\end{equation}
+in the same fashion we can transform the expression $(b)$ getting the following result
+\begin{equation}
+\begin{split}
+& \int_{-\infty}^{\infty}\chi(\tau_i)\chi(\tau_j)Q_s(\tau_1,\tau_2,...,\tau_s)d\tau_1d\tau_2...d\tau_s \\
+& \Leftrightarrow \int_{t_a}^{t_b}\left[d\tau_1\int_{t_a}^{t_b}d\tau_2\int_{-\infty}^{\infty}Q_s(\tau_1,\tau_2,...,\tau_s)d\tau_3...d\tau_s\right] \\
+\end{split}
+\end{equation}
+in the end we can rewrite $\langle N^2 \rangle$
```
Comment (DanielSank) [inline chapter2/Mean_square_N_interval.tex]: Capitalize beginning of new sentence?

File: chapter2/Description_sequence_random_events.tex [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+Q_s(\tau_1, \tau_2,...,\tau_s)=P(\tau_1)P(\tau_2)...P(\tau_s)
+\end{equation}
+Thus we have
+\begin{equation}
+\begin{split}
+& P(\tau_1)= w(\tau_1) \\
+& P(\tau_2)= \int_{0}^{\tau_2}w(\tau_2-\tau_1)d\tau_1\\
+& P(\tau_3)=  \int_{0}^{\tau_3}w(\tau_3-\tau_2)d\tau_2\\
+& ... \\
+& P(\tau_s)=  \int_{0}^{\tau_s}w(\tau_s-\tau_{s-1})d\tau_{s-1}\\
```
Comment (DanielSank) [inline chapter2/Description_sequence_random_events.tex:18]: Just FYI, in TeX you can do, for example, `\int_0^\infty`, i.e. you don't need braces in superscripts and subscripts if there's only one symbol.

**Review by DanielSank — CHANGES_REQUESTED** [review]
Thanks, this is great. There are a couple of little punctuation errors. If you can fix them up I'll merge this.

**Comment by Edlingeer** [comment]
Hi,

No problem, I'll fix this after work today. And thanks for the feedback.

--
Thomas

--
Thomas

2017-03-06 9:39 GMT+01:00 Daniel Sank <notifications@github.com>:

**Comment by Edlingeer** [comment]
I made some changes as you asked. Is it okay now ? sorry for the time it took me to get to this, I had some unexpected last minutes things that came up.

**Comment by DanielSank** [comment]
Woah, I'm sorry I forgot about this. I'll try to look at it this weekend.

**Review by DanielSank — DISMISSED** [review]
Thanks for your contribution. I'm merging as is and will make a follow-up commit to make this new content adhere to the style guide.

**Review by DanielSank — APPROVED** [review]

**Pull request merged**